src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -918,6 +918,12 @@ public boolean isSuppressedEmbeds()
         return MessageFlag.fromBitField(flags);
     }
 
+    @Override
+    public boolean isEphemeral()
+    {
+        return (this.flags & MessageFlag.EPHEMERAL.getValue()) > 0;
```

Comment:
```suggestion
        return (this.flags & MessageFlag.EPHEMERAL.getValue()) != 0;
```

Reply:
I don't get why it should be != instead of >, i copied the style of `isSuppressedEmbeds`

```java
@Override
public boolean isSuppressedEmbeds()
{
  return (this.flags & MessageFlag.EMBEDS_SUPPRESSED.getValue()) > 0;
}
```